lib/graphiti/rails/railtie.rb

```diff
@@ -2,27 +2,7 @@
 
 module Graphiti
   module Rails
```

Comment:
You thinking the Railtie in Graphiti goes here?

Reply:
Yeah